### DISCUSSION TITLE
Cannot reach admin console in production mode, start-dev works

### DISCUSSION THREAD

**Discussion by notTrinity** [keycloak/keycloak#15230] [Archive]
docker entry:
exec /opt/keycloak/bin/kc.sh start-dev --import-realm --https-port=8563 --db postgres --db-password platform --db-username platform \
     --db-url-host ${KC_DB_IP}  --db-url-port ${KC_DB_PORT} --db-url jdbc:postgresql://${KC_DB_IP}:${KC_DB_PORT}/demodb \
     --https-trust-store-password ${TRUSTSTORE_KEY_PASS} --https-key-store-password ${INTERNAL_KEYSTORE_PASSWORD} &

Works and I can reach the admin console,
Just changing it to start I, there is an error step1.html 

status (failed) net::error connection refused.

**Comment by Carmezim** [comment]
Would be helpful for others to help if you can post logs.

**Comment by notTrinity** [comment]
I did some more digging and found that I had to configure a few more things.
I thought I had done all the ssl stuff but missed some in here:
https://www.keycloak.org/server/configuration-production
